katnip/legos/http.py

```diff
@@ -42,6 +42,15 @@ def __init__(self, key, value, end=False, fuzzable_key=False, fuzzable_value=Tru
         super(IntField, self).__init__(key, value_field, end, fuzzable_key, fuzzable_value)
 
 
+class ContentLengthField(CustomHeaderField):
+    def __init__(self, value, sized_field, end=False, fuzzable_key=False, fuzzable_value=True):
+        value_field = OneOf(fields=[
+            Size(sized_field=sized_field, length=32, encoder=ENC_INT_DEC),
+            DecimalNumber(name="value", value=value, num_bits=32, signed=True),
+        ])
+        super(ContentLengthField, self).__init__('Content-Length', [value_field], end, fuzzable_key, fuzzable_value)
```

Comment:
No need to set value_field in a list, Container (and CustomHeaderField) can handle a single field as well

Reply:
Realy! I didn't known. That's great